src/srptn/common/data/entities/analysis.py

```
@@ -135,15 +135,16 @@ def export_metadata(self) -> None:
         with (self.meta_path / "details.yml").open("w") as f:
             yaml.safe_dump(details, f)
 
-    def get_config(self) -> dict:
+    def get_config(self) -> dict | None:
```

Pylint (3.3.7)

[refactor] Either all return statements in a function should return an expression, or none of them should.

(R1710)